docs/repositories-coverage/commits.md

```diff
@@ -76,3 +76,19 @@ The option **Show also files without coverage changes** allows you to list all f
 ![Files tab](images/{{ page.meta.file_name }}-tab-files.png)
 <!-- vale on -->
 <!--tab-files-end-->
+
+<!--tab-diff-start-->
+## Diff tab
+
+The **Diff** tab displays a line-by-line view of the coverage variation introduced by the {{ page.meta.page_name }}. It includes the following areas:
+
+-   A list of files modified by the {{ page.meta.page_name }}, showing next to each file either the total file coverage or the coverage variation introduced by the {{ page.meta.page_name }}.
```

Comment:
> showing next to each file **either** the total file coverage or the coverage variation

It's not clear when we show the total file coverage or the coverage variation... Is the yellow/green icon related? Maybe we can explain what these colors mean.

Reply:
We show the total file coverage when the commit/PR introduces no variation. Could this work?

```suggestion
-   A list of files modified by the {{ page.meta.page_name }}, showing next to each file either the coverage variation introduced by the {{ page.meta.page_name }}, or the total file coverage if the {{ page.meta.page_name }} introduces no coverage variation.
```

On a side note, the icon colors show if the file is added (green)  or modified (yellow). Should we clarify that as well? Thoughts?